api/hooks/customGameHook/index.js

```diff
@@ -3,11 +3,13 @@ module.exports = function gameHook() {
   // Game API //
   //////////////
   return {
-    createGame: function (gameName) {
+    createGame: function (gameName, isRanked = false) {
       return new Promise(function (resolve, reject) {
+        const ranked = isRanked ?? false;
         Game.create({
           name: gameName,
           status: true,
+          ranked,
```

Comment:
That said, you may want to use it directly and coerce it to be a boolean though.
```suggestion
          ranked: ranked === true,
```